assets/src/application/ui/layout/entityList/pagination/EntityPagination.tsx

```diff
@@ -8,23 +8,20 @@ import './style.scss';
  * Adds pagination to an "EntityList" component
  * @return EntityPagination
  */
-const EntityPagination = ({
-	current,
-	onChange,
-	pageSize = 10,
-	pageSizeOptions = ['6', '12', '24', '48'],
+const EntityPagination: React.FC<PaginationProps> = ({
+	pageSizeOptions = ['2', '6', '12', '24', '48'],
```

Comment:
@tn3rb I noticed `2` being used in filterBar control, that's why I've added it here